.github/pull_request_template.md

```diff
@@ -1,6 +1,16 @@
-## Description
+## What is the change?
+
+<!-- MANDATORY: Include a summary of your change -->
+
+## Why is the change being made?
+
+<!-- MANDATORY: And Link to any related GitHub Issues -->
+
+
+
+<!-- Optional: Link to any related GitHub Issues -->
```

Comment:
```suggestion

```

Reply:
Not sure what you intended here, but suggesting some cleanup

Reply:
oh!

in the `mandatory` above it also mentions Github  Issue links, and I was just deleting one. But this is fine, reject this and clean up the other one!